article/sections/_2_graph_importance.tex

```diff
@@ -104,31 +104,30 @@ \section{Perils of disregard}
 The goal of our simulation exercise is to show that ignoring the true generative model of the data (Figures \ref{fig:DA_causal_2} through \ref{fig:SR3_causal_2}) can result in arbitrarily biased treatment effects, even if the analyst knows the true outcome model (Equations \ref{eq:DA} through \ref{eq:SR3}).
 
 \begin{figure}
-   \centering
+\centering
+\begin{subfigure}
```

Comment:
This was the breaking change for the pdf.

When using subcaption, we need to put the width commands next to `\begin{subfigure}` not next to `includegraphics`. See page 6: http://mirrors.ibiblio.org/CTAN/macros/latex/contrib/caption/subcaption.pdf